script/test.sh

```diff
@@ -82,16 +82,19 @@ test_rebase_abort () {
    git add file-11
    git commit -m "Add file-11"
 
+   echo "Add some modification" >> file-11
+   HAS_UNMODIFIED_FILE=$(git status --short | grep file-11 | wc -l)
```

Comment:
In the current implementation, `HAS_UNMODIFIED_FILE` has always "1" and the following condition `-a "${HAS_UNMODIFIED_FILE}" != "0"` returns always "true".

`HAS_UNMODIFIED_FILE=$(git status --short | grep file-11 | wc -l)` should be after `./"${EXEC_COMMAND}" -n 2..5 -f -d` line.

Reply:
@chaspy You are right. I fixed it.